main/zoe/guide/README.md

```diff
@@ -78,7 +78,7 @@ the swap contract behaves badly, we will both get a refund, and at
 best, we'll get what we each wanted.
 
 Let's look at the basic `publicSwap` contract ([full text of
-the real contract](https://github.com/Agoric/ERTP/blob/master/core/zoe/contracts/publicSwap.js)).
+the real contract](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/contracts/publicSwap.js)).
 
 Here's a high-level overview of what would happen:
 1. I make an instance of the swap contract.
```

Comment:
These steps have changed with the new Zoe. Instead of getting an escrow receipt, we get an invite, and we can send that invite to the other user.